google/cloud/storage/client_options.h

```diff
@@ -26,12 +26,12 @@ inline namespace STORAGE_CLIENT_NS {
 /**
  * Describes the configuration for a `storage::Client` object.
  *
- * By default read several environment variables to configure the client:
+ * By default, several environment variables are read to configure the client:
  *
  * - `CLOUD_STORAGE_TESTBENCH_ENDPOINT`: if set, use this http endpoint to
  *   make all http requests instead of the production GCS service. Also,
- *   if set it uses the `google::cloud::storage::oauth2::AnonymousCredentials`
- *   by default.
+ *   if set, the CreateDefaultClientOptions() function will use an
+ *   AnonymousCredentials object instead of loading ADCs
```

Comment:
I believe we should not assume that the customers know what `ADC` stands for, it should be defined the first time it is used in a class documentation.

Reply:
Fixed.